### ISSUE TITLE
cast to TV / DLNA casting

### ISSUE THREAD

**Issue by glcparma** [fsmdeveloper/hdo-box#9]
it would be great to be able to cast to TV from the mobile, is there any plan to implement this feature? thank you

**Comment by fsmdeveloper** [comment]
Yes, this feature will be implemented very soon. I'm currently working on it. Thank you!

**Comment by bizdatech** [comment]
thx for your projecct i hope this is it last long project, 

poor people like us cant afford all netflix ,disney and many more. 

if got add korean and chinese tv series would be nicer. 

like this yogurt tv , but that one need to purchase their tvbox :( which is like around 200USD

anyway havent try yet to my tv, will test it out later. comment first :) to show the appreciatre someone created this apk

**Comment by fsmdeveloper** [comment]
Thank you

**Comment by peptech-qais** [comment]
Loving the app. I would like to second this specific request. Ability to cast to devices would surely make it much more usable with a well rounded experience.

**Comment by peptech-qais** [comment]
Loving the app. I would like to second this specific request. Ability to cast to devices would surely make it much more usable with a well rounded experience.

**Comment by qais1995** [comment]
Loving the app. I would like to second this specific request. Ability to cast to devices would surely make it much more usable with a well rounded experience.
